agenta-backend/agenta_backend/tasks/evaluations.py

```diff
@@ -338,8 +338,13 @@ async def aggregate_evaluator_results(
                 )
 
             else:
-                # Handle boolean values for auto_regex_test and other evaluators
-                if all(isinstance(result.value, bool) for result in results):
+                if evaluator_key == "auto_regex_test" and all(
+                    isinstance(result.value, bool) for result in results
+                ):
+                    average_value = sum(result.value for result in results) / len(
```

Comment:
isn't this the same line of code as below??

Reply:
The difference lies in the second condition:

```python
all(isinstance(result.value, bool) for result in results)
```

This condition is specifically for evaluating a webhook and checking if all the values in the result are booleans. If so, it then proceeds to run the aggregation. 

The right way to do the aggregation after the check is to convert the boolean into an integer:

```python
average_value = sum(int(result.value) for result in results) / len(results)
```

I'll work on this now.